### PR TITLE
[DOCS-6423] Add PATH setup

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: search-enterprise/latest/using/path.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The PATH field is the Elasticsearch field which contains the `primaryHierarchy` attribute that consists of a list of `noderefs`. Field indexing is different for re-indexing and live indexing components because they rely on different sources of information. The setup sequence for new systems is:
+
+1. RDBMS (re-indexing)
+
+2. Events (live indexing)
+
+The live indexing component populates the PATH field on Elasticsearch documents starting from the `primaryHierarchy` attribute found in the node event. The `primaryHierarchy` attribute captures the primary hierarchy of ancestors of the resource affected, which means the folder path of the content. The first element is the immediate parent. For example this is a node event which contains that information:
+
+```JOSN
```
Comment (mjstanford) [inline search-enterprise/latest/using/path.md:13]: Typo?

**Review by mjstanford — COMMENTED** [review]
PATH query documentation should also include removal of PATH queries from the unsupported features list

**Review by oliveralfresco2018 — COMMENTED** [review]
LGTM

**Review by gravitonian — APPROVED** [review]
Built and read through the new path queries page. LGTM

**Pull request merged**